src/renderer/remote-loader.ts

```diff
@@ -115,13 +126,40 @@ export class RemoteLoader {
     }
   }
 
+  public async getGistRevisions(gistId: string): Promise<GistRevision[]> {
+    console.log('getGistRevisions!');
```

Comment:
suggestion: should we remove this log line if we already have the response logged below?